### PR TITLE
update brightness.url when color is set

### PR DESCRIPTION
**Pull request by its2loud**
make use of color.brightness setting to switch between brightness.url or hsl method.
• set color.brightness to false or unset to update brightness.url
• set color.brightness to true to update HSL as RGB

_setRGB seems to change the saturation instead of the brightness though, but i have not investigated that further.
(see issue #14)

### CODE REVIEW

**Comment by jnovack** [comment]
I'm not sure I want Switch and Lock in this.  Preferably, there would be other homebridge plugins for Switch and Lock, that is the nature of plugins, why have 1 monolithic plugin when you can have multiple.

Would you mind disabling Switch and Lock?

**Comment by its2loud** [comment]
I already disabled Lock type. It's very useful for my purpose to be able to chose Switch as a more generic type for devices that only support on and off.

File: index.js [hunk 1]
```diff
@@ -184,7 +185,7 @@ HTTP_RGB.prototype = {
                     .on('set', this.setLockTarget.bind(this));
 
                 return [lockService];
-
+
```
Comment (jnovack) [inline index.js]: Extra Spaces.

**Review by jnovack — CHANGES_REQUESTED** [review]
Please make two changes.

1) Destroy those extra spaces.
2) Version bump to v1.0.2.

thanks!

File: index.js (hunk 1 above)
Comment (its2loud) [reply index.js]: Done, but I'm not totally sure if I got it. 
Please check if the changes are like you requested.

**Review by jnovack — APPROVED** [review]

**Pull request merged**